drltrace/drltrace.c

```diff
@@ -161,7 +161,8 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
             dr_fprintf(outf, "<invalid memory>");
             /* Just keep going */
         });
-        dr_fprintf(outf, ")");
+        dr_fprintf(outf, ") thread id: "TIDFMT" and return to "PFX"",
```

Comment:
> "How about using the thread id as a prefix, like DrM does"
> Do you mean this prefix:
> ~~Dr.M~~ for example ~~2953~~

(There's a "quote" button to quote pasted text btw.)

Yes, a prefix at the start of each line identifying the thread.  Other tools use this format (e.g., Valgrind) so users may expect it.